### PR TITLE
Make desktop name configurable from browser-laptop

### PR DESCRIPTION
**Pull request by darkdh**
Auditors: @bsclifton, @bbondy

### CODE REVIEW

File: atom/browser/browser_linux.cc [hunk 1]
```diff
@@ -118,6 +123,12 @@ bool Browser::SetAsDefaultProtocolClient(const std::string& protocol,
 // |protocol|.
 bool Browser::IsDefaultProtocolClient(const std::string& protocol,
                                       mate::Arguments* args) {
+  std::string desktop_name;
+  if (!args->GetNext(&desktop_name)) {
```
Comment (posix4e) [inline atom/browser/browser_linux.cc]: Should we check to see if it's empty?

Comment (darkdh) [reply atom/browser/browser_linux.cc]: The empty desktop name does no harm. For IsDefaultProtocolClient, the final result will be false and SetAsDefaultProtocolClient would be no effect.

**Comment by bridiver** [comment]
please mark with branch labels/milestone

**Comment by bsclifton** [comment]
should be merged for browser-laptop 0.20.x (must happen at same time as https://github.com/brave/browser-laptop/pull/10552 )

**Pull request merged**